source/views/settings/screens/debug/list.js

```diff
@@ -44,14 +49,23 @@ export class DebugListView extends React.PureComponent<Props> {
 	)
 
 	render() {
-		let keyed = toPairs(this.props.state).map(([key, value]) => {
+		let nothingFoundMessage =
+			this.props.testing || this.props.navigation.getParam('testing')
+				? 'Nothing parsed.'
+				: 'Nothing found in redux.'
+
+		let parsedData = this.props.testing
+			? JSON.parse(this.props.state)
+			: this.props.state
+
+		let keyed = toPairs(parsedData).map(([key, value]) => {
 			return {key, value}
 		})
 
 		return (
 			<FlatList
 				ItemSeparatorComponent={ListSeparator}
-				ListEmptyComponent={<NoticeView text="Nothing found in redux." />}
+				ListEmptyComponent={<NoticeView text={nothingFoundMessage} />}
```

Comment:
```suggestion
				ListEmptyComponent={<NoticeView text="Nothing found." />}
```

maybe? (I don't think we need a distinction between the two modes here.)